scripts/travis_install

```diff
@@ -26,6 +26,7 @@ conda install --yes $DEPS_TRAVIS
 conda config --add channels bokeh
 conda config --add channels javascript   # TODO: move packages to bokeh?
 conda config --add channels mutirri   # TODO: remove when packages are moved to bokeh
+conda config --add channels jrderuiter   # TODO: remove when pytest-cov available on bokeh channel
```

Comment:
https://anaconda.org/hargup/pytest-cov/files, seems to have a noarch package... we can probably test that or build one for py33